resources/reproducing-ci.md

```diff
@@ -108,13 +110,14 @@ In RAPIDS CI workflows, the builds and tests occur on different machines.
 
 Machines without GPUs are used for builds, while the tests occur on machines with GPUs.
 
-Due to this process, the artifacts from the build jobs must be downloaded from [downloads.rapids.ai](https://downloads.rapids.ai) in order for the test jobs to run.
+Due to this process, the artifacts from the build jobs must be downloaded from GitHub Artifacts in order for the test jobs to run.
 
 In CI, this process happens transparently.
 
 Local builds lack the context provided by the CI environment and therefore will require input from the user in order to ensure that the correct artifacts are downloaded.
 
-Any time the `rapids-download-conda-from-s3` command (e.g. [here](https://github.com/rapidsai/cugraph/blob/b50850f0498e163e56b0374c1c64e551a5898f26/ci/test_python.sh#L22-L23)) is encountered in a local test run, the user will be prompted for any necessary environment variables that are missing.
+Any time the `rapids-download-conda-from-github` command (e.g. [here](https://github.com/rapidsai/cugraph/blob/b50850f0498e163e56b0374c1c64e551a5898f26/ci/test_python.sh#L22-L23)) is encountered in a local test run, the user will be prompted for any necessary environment variables that are missing.
```

Comment:
This link probably needs to be updated to a commit with the new commands. The todo comment below can be removed.